packages/router/src/__tests__/analyzeRoutes.test.tsx

```diff
@@ -653,7 +653,7 @@ test('Nested sets, and authentication logic', () => {
   })
 })
 
-test('Give correct ids to root sets', () => {
+test('gives correct ids to root sets', () => {
```

Comment:
I actually think this was better the way it was.
Had this been an `it`-test instead I would have agreed with your change. But with `test` I think "test: Give correct..." reads better

(But I really wish all tests used `it()`)